src/Microsoft.DotNet.Build.Tasks/project.json

```diff
@@ -1,8 +1,27 @@
 {
   "dependencies": {
-    "NuProj": "0.10.4-beta-gf7fc34e7d8"
+    "NuProj": "0.10.4-beta-gf7fc34e7d8",
+    "Microsoft.Build.Framework": "0.1.0-preview-00005",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
+    "Microsoft.CSharp": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Xml.XDocument": "4.0.0",
+    "System.IO.Compression": "4.0.0",
+    "System.IO.Compression.ZipFile": "4.0.0",
+    "System.Xml.ReaderWriter": "4.0.10",
+    "System.Xml.XPath.XmlDocument": "4.0.0",
+    "System.Reflection": "4.0.10",
+    "System.Reflection.Metadata": "1.1.0-alpha-*",
+    "Newtonsoft.Json": "7.0.1"
   },
   "frameworks": {
```

Comment:
We may want to have a Net46 target as well, but that won't work well with frameworks + RIDs because it results in a cross-product of all frameworks x all RIDs.  Once we switch to using nuget.exe/dotnet.exe to do the restore in buildtools then this should be built as a v3 portable library with "framework": { "dotnet5.4" } and "supports" clauses for each TFM:RID tuple we want to target.